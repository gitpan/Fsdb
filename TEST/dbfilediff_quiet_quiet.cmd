prog='dbfilediff'
args='-E --quiet --quiet --input TEST/dbfilediff_epsilon_cases.in-2'
in=TEST/dbfilediff_epsilon_cases.in
cmp='diff -cb '
