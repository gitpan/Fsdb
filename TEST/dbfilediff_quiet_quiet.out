#fsdb x diff
5.23	-
5.2	+
8.23	-
8.25	+
9.23	-
9.20	+
#   | dbfilediff -E --quiet --quiet --input TEST/dbfilediff_epsilon_cases.in-2
